### PR TITLE
highlight today cell

### PR DESCRIPTION
**Pull request by LohithAnand**

### CODE REVIEW

File: angular-material-calendar.css [hunk 1]
```diff
@@ -50,3 +50,5 @@ calendar-md {
       calendar-md md-content > md-content.calendar > :not(:first-child) > * :last-child {
         overflow: hidden;
         text-overflow: ellipsis; }
+      calendar-md md-content > md-content.calendar .currentDayHighlight {
+          background-color: #FCF8CD !important; }
```
Comment (gavinbarron) [inline angular-material-calendar.css]: Can we please not use the !important here? I don't see anything else setting the background color property so there should be no need for using !important.

Comment (LohithAnand) [inline angular-material-calendar.css]: That's a good catch G. !important is not required.

**Comment by gavinbarron** [comment]
Thanks!

**Comment by ybelouad** [comment]
The today cell is no more highlighted :-(
It seems that the commit 2155fc3a5fbcabfaeef9fd8f7322fdf4bbb51368  has deleted the isCurrentDay function.

**Comment by theantichris** [comment]
@gavinbarron Are there plans to get isCurrentDay back in?

**Pull request merged**